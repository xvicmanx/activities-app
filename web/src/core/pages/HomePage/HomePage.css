.HomePage {
  padding: 15px;
  margin-bottom: 200px;
}

.HomePage__Title {
  margin-bottom: 0;
  padding-left: 10px;
  color: #767e84;
  font-size: 1.5rem;
}

.Graph {
  max-width: 585px;
  background: #fff;
  padding: 20px;
  margin: 20px 10px;
}

.Graph__Title {
  color: #767e84;
  font-size: 1.2rem !important;
}

.Graph .axis {
  font-size: 12px;
}

.Graph .table thead th, .Graph .table {
  color: #767e84;
}

@media only screen and (min-width: 1024px) {
  .HomePage__Content {
    display: flex;
  }

  .HomePage__Title {
    font-size: 1.8rem;
  }
}