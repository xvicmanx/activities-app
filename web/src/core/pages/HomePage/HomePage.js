import React from 'react';
import { Heading, Image, Message } from 'react-bulma-components';

import homeIcon from "../../../assets/images/front.svg";

import './HomePage.css';

const HomePage = () => (
  <div className="HomePage">
    <Heading className="HomePage__Title">
      <Image
        src={homeIcon}
        alt="home"
        className="Title-Icon"
      /> Inicio
    </Heading>
    <div className="HomePage__Content">
      <div>
        <Message color="info">
          <Message.Body>
            Bienvenido!
          </Message.Body>
        </Message>
      </div>
    </div>
  </div>
);

export default HomePage;
