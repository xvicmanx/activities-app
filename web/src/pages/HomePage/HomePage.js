import React from 'react';
import { Heading, Image } from 'react-bulma-components';

import homeIcon from "../../assets/images/front.svg";

import './HomePage.css';

const HomePage = () => (
  <div className="HomePage">
    <Heading className="HomePage__Title">
      <Image
        src={homeIcon}
        alt="home"
        className="Title-Icon"
      /> Inicio
    </Heading>
    <div className="HomePage__Content">
      <div>
        Hello
      </div>
    </div>
  </div>
);

export default HomePage;
