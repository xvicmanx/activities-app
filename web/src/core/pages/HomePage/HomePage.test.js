import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { faBuilding } from '@fortawesome/free-solid-svg-icons';

import HomePage from './HomePage';

describe('HomePage', () => {
  it('renders properly', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<HomePage />);
    expect(result).toMatchSnapshot();
  });
});
