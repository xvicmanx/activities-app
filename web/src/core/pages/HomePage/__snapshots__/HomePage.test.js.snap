// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders properly 1`] = `
<div
  className="HomePage"
>
  <Heading
    className="HomePage__Title"
    heading={false}
    renderAs="h1"
    spaced={false}
    subtitle={false}
  >
    <Image
      alt="home"
      className="Title-Icon"
      fallback="https://bulma.io/images/placeholders/480x480.png"
      rounded={false}
      src="front.svg"
    />
     Inicio
  </Heading>
  <div
    className="HomePage__Content"
  >
    <div>
      <Message
        color="info"
        renderAs="article"
      >
        <MessageBody
          renderAs="div"
        >
          Bienvenido!
        </MessageBody>
      </Message>
    </div>
  </div>
</div>
`;
